spec/setup_test_database.rb

```diff
@@ -0,0 +1,7 @@
+require 'pg'
+
+def setup_test_database
+  p "Setting up test database..."
+  connection = PG.connect(dbname: 'chitter_test')
```

Comment:
these are two different databases - you need to truncate/clear the one that the tests are being run on, so chitter_test